src/main/resources/application.yaml

```
@@ -15,9 +17,21 @@ management:
       exposure:
         include: health, info
 
+url:
+  fellesdatakatalog.api: https://resource.api.fellesdatakatalog.digdir.no
+  fellesdatakatalog.frontend: https://data.norge.no
+
 ---
 spring:
-  config.activate.on-profile: test
+  config.activate.on-profile: dev
 
+logging:
+  level.no.digdir.accessrequestapi: DEBUG
 
+url:
+  fellesdatakatalog.api: https://resource.api.staging.fellesdatakatalog.digdir.no
+  fellesdatakatalog.frontend: https://staging.fellesdatakatalog.digdir.no
```

Hvordan gjør vi det så applikasjonen vet den skal gå hit i staging? Pleier dere å bruke Server-side discovery eller er det noen miljøvariabler som kan trigge en egen profil i spring?

---

Vi legger alt som varierer mellom miljøene inn som miljøvariabler
noe ala dette, der `RESOURCE_SERVICE_URI` og `FDK_URI` blir definert ut fra secrets i deploy-filene
```suggestion
  fellesdatakatalog.api: ${RESOURCE_SERVICE_URI}
  fellesdatakatalog.frontend: ${FDK_URI}
```

---

Skjønner. Hvordan setter jeg miljøvariablene? Er jeg på riktig spor i [Set urls in env variables](https://github.com/fellesdatakatalog/access-request-api/pull/5/commits/e3dca38132ac79581800dfe5d5eb7cd0fca2809c)?

Ser deploy feiler https://github.com/fellesdatakatalog/access-request-api/actions/runs/11257485578/job/31301954617, men mistenker at det ikke er endringen jeg la til siden feilmeldingen er `gcloud: command not found`

---

for direkte angivelse i deploy-steget så blir det korrekt, men la til noen kommentarer på hvordan du kan få tak i de verdiene via våre eksisterende secrets. Du skal ikke måtte gjøre noen endringer i workflows-mappa, bare deploy.

Dunno hva som skjer med action som feiler, kanskje den `build_env: true` har noe med saken å gjøre? @valosnah du vet kanskje?

---

Ops  den `build_env` er fra meg som prøvde meg litt fram. Fjerner det i tillegg til at jeg endrer miljøvariablene til å hente dem fra `secretKeyRef` her: https://github.com/fellesdatakatalog/access-request-api/pull/5/commits/5e036a706fbab8d8e44037ef5690946c869b5eb3